components/admin/lessons/AdminLessonInputs.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react'
+import { useAddModuleMutation } from '../../../graphql'
+import { formChange } from '../../../helpers/formChange'
+import { FormCard, Option } from '../../FormCard'
+import { AlertFillIcon, CheckCircleIcon } from '@primer/octicons-react'
+import styles from '../../../scss/adminLessonInputs.module.scss'
+import { Spinner } from 'react-bootstrap'
+import { GraphQLError } from 'graphql-request/dist/types'
+
+type Props = {
+  lesson: { title: string; id: number }
+}
+
+enum Error {
```

Comment:
Is there potential for more error messages to be added in the future? If so, making an enum is fine. If not, then it would be more appropriate to declare this as a regular const variable instead of enum. If there no potential to add more items to the enum list, then declaring an enum(set of constants) for just one item(`INVALID_DATA`) is a readability issue since it will increase complexity without good reason

Reply:
I think there is potential for more error messages to be added to the future for this file, but I just wanted to confirm this with you since you have more context than me

Reply:
There will be more errors from the GraphQL resolver after we update it to validate the arguments (it currently doesn't). For example, checking if the module name consists of at least alphanumeric characters or the description satisfies the max length.